### PR TITLE
Make TF dataset build for 2.3.0

### PR DESCRIPTION
**Pull request by awolant**
Signed-off-by: Albert Wolant <awolant@nvidia.com>

#### Why we need this PR?
 - It adds support for TF 2.3.0
 - Updates other TF versions

**JIRA TASK**: *[DALI-1513]*

### CODE REVIEW

**Comment by awolant** [comment]
!build

File: qa/setup_packages.py [hunk 1]
```diff
@@ -324,7 +324,8 @@ def get_pyvers_name(self, url, cuda_version):
                         { "100" : [
                               PckgVer("1.15.2",  python_max_ver="3.7"),
                               PckgVer("2.1.1",  python_max_ver="3.7"),
-                              "2.2.0"]
+                              "2.2.0",
+                              "2.3.0"]
```
Comment (awolant) [inline qa/setup_packages.py:328]: With regards to this, what do you think about dropping tests for 2.1.1? And possibly moving from 1.15.2 to 1.15.3? Almost no difference there, but it is the latest version.

Comment (JanuszL) [reply qa/setup_packages.py:328]: I'm for both ideas.

**Comment by awolant** [comment]
!build

File: qa/setup_packages.py (hunk 1 above)
Comment (awolant) [reply qa/setup_packages.py]: Ok, done.

**Pull request merged**